xds/src/main/java/io/grpc/xds/EdsLoadBalancer.java

```diff
@@ -328,10 +368,17 @@ public void shutdown() {
 
     @Override
     public void onEndpointChanged(EndpointUpdate endpointUpdate) {
-      channelLogger.log(
-          ChannelLogLevel.DEBUG,
-          "EDS load balancer received an endpoint update: {0}",
-          endpointUpdate);
+      logger.log(XdsLogLevel.DEBUG, "Received endpoint update: {0}", endpointUpdate);
```

Comment:
Override `toString()` for `EndpointUpdate`

Reply:
It is already overridden.